docs/Introduction/tutorials/beginners-tutorial.md

```diff
@@ -115,34 +99,18 @@ Another familiar concept to programmers is the **constructor**. It is called upo
 In `HelloWorld`, the constructor takes in a `string` as a parameter and sets the `message` state variable to that string.
 
 ```solidity
-pragma solidity 0.8.7;
-
-contract HelloWorld {
-  constructor(string memory initialMessage) {
-        message = initialMessage;
-  }
-}
+{% include snippets/Tutorials/beginner-constructor.sol %}
```

Comment:
Minimizing this example is fine, but I don't think it needs to be in the snippets folder.